### DISCUSSION TITLE
Assignment of first authors

### DISCUSSION THREAD

**Discussion by ajshajib** [AstroBridge/BDLensing#208] [Announcements]
I am assigning a few of the participants to take responsibility for being included in the first-author group of the paper. The first authors would be recognized in the paper for having equal contributions at the top level, as done in this paper: https://arxiv.org/pdf/2208.02233.pdf. The first authors can indicate in their CV that they are joint first-authors of the paper. All the other participants with significant contributions will be added as co-authors, and their author ordering will depend on their amount of contribution. We will strive for fair recognition of all the contributors using the [CRediT system](https://www.elsevier.com/authors/policies-and-guidelines/credit-author-statement).

The responsibilities of the people in the first-author group would be:

1. **Managing lens models:** to ensure the models/final deliverables are completed by the deadline. The modeling manager should check in with the original modeler and help the modeler as necessary to get them to the finish line. The manager should also ensure the relevant PRs are progressing in a timely manner through the review process. If the original modeler of the system is not actively participating anymore, or if they can't meet the deadline, then the manager may have to finish the modeling him/herself unless any other participant is available to take on the system. The deadline to finish the MCMC issues for the models would be **Sept 30** and the deadline to finish the final deliverable step is **Oct 15**.
2. **Take care of the final analysis for the whole sample:** for example, to measure the Pearson correlation coefficient between $\Sigma_{10}$ and the offsets in the centroid and position angle between the mass and light of the lensing galaxy.
3. **Write different sections** of the paper and review/edit sections written by others. This step should be completed, and the paper to be submitted on arXiv by **Nov 30**.
4. **Respond to the referee:** the referee report of the submitted paper will be received 2-3 months after the submission, so around February. Depending on the report, the first authors would need to do additional work around February-April 2024. The first authors have committed to be available for additional work in that period.

So far, @JobairHasan, @rafee-adnan, @Rajit13, @aaimtiaz, @SulymanHossain, and @ZareefJafar have expressed their intent to be recognized in the first author group. I initially reached out to @rafee-adnan, @Rajit13, @aaimtiaz, @SulymanHossain based on [their high number of PR reviews](https://github.com/ajshajib/BDLensing/actions/runs/6011508219), and to @Rajit13 and @JobairHasan based on their advanced completions of lens modeling assigned to them. Additionally, @ZareefJafar expressed intent to be recognized as the first author during a telecon. All these participants are also regular attendees of the weekly telecons. **If anyone else from the participants would like to express intent to be recognized as a first author, please do so by tomorrow in reply to this thread.**  However, all of these participants are currently **candidates** to be in the first-author group. The inclusion within the first author group would be finalized just before the submission of the paper, and those who have carried out **all of the above responsibilities** equally as other first authors would be included in the first author group. _So, if you have expressed intent now, your overall contribution to the project needs to be at a similar level as other top contributors to be confirmed as a first author at the end of the project._

However, as stated above, everyone's order on the author list would depend on their contribution at the end of the project. So even if someone is not recognized as a first author at the end, their name can be higher in the order list depending on how much of the above responsibilities they have carried out.

**Comment by mamun-nahid** [comment]
Can I get recognized as a first author? I have already completed my MCMC run and created a pull request. I think I can complete the other sections of my system within the deadline and manage some other lens systems too.

**Reply by ajshajib** [reply]
You are most welcome to act as managers for some other systems, and good job on your advanced progress on lens modeling.  However, to confirm with you, your overall contribution needs to be at the same level as other top contributors at the end of the project. So, finishing only one lens model will not be enough to match with other top contributors. So far, you have not reviewed any of the PRs. Additionally, managing a system may require you to finish the lens modeling yourself if the original modeler is progressing slower than expected. So, you are most welcome to join as a candidate first author like the others, but you should work on carrying out **all of the four responsibilties** toward making your overall contribution to match at a similar level with the other top contributors, and join the telecons regularly from now onward. Please confirm that you are making a commitment to carrying out these tasks, and then I will assign you as managers for some systems.

**Reply by mamun-nahid** [reply]
Okay bhaia. I will carry out the responsibilities you've mentioned properly. I will also be more regular in attending meetings and participating in reviewing PRs.

**Reply by ajshajib** [reply]
Ok, great! I have assigned you as managers for a couple of systems.

**Reply by mamun-nahid** [reply]
Thank you bhaia.

**Comment by TwistNemo** [comment]
Could I kindly request recognition as the first author? I've completed my MCMC run and will soon submit a pull request, and I'm eager to continue contributing

**Reply by ajshajib** [reply]
You are most welcome to be considered as a candidate to be included in the first-author group. Just to clarify, the first-author group is the people whose names will appear at the top of the author list. Everyone else who has substantially contributed to this project is already considered a co-author, and their order in the author list will be based on the amount of contribution. Your contribution so far qualifies as substantial for this co-authorship.

On the point of you being included in the first-author group, I will reiterate some of the points I told mamun-nahid above. The first author group, which will be decided at the end of the project, will include the participants who have contributed the most and at a similar level to each other. Although you have done great so far on your own model, your overall contribution level currently is not matching with the top contributors so far. For example, you have yet to review any PR from others, and I haven't seen you attend the weekly telecons and join the discussion recently. So, my suggestion for you to catch up with the top contributors would be to finish up your model as soon as possible, actively participate in reviewing others' PRs, be regular at the weekly telecons, and look for an opportunity to model another lens system for which the original modeler stopped working on or needs help.
